view/game_configuration_panel.py

```diff
@@ -790,11 +789,19 @@ def autodetect_paths_by_platform(self) -> None:
             f"/Users/{getpass.getuser()}/Library/Application Support/Rimworld/Config/",
             f"/Users/{getpass.getuser()}/Library/Application Support/Steam/steamapps/workshop/content/294100/",
         ]
-        linux_paths = [
-            f"{expanduser('~')}/.steam/debian-installation/steamapps/common/RimWorld",
-            f"{expanduser('~')}/.config/unity3d/Ludeon Studios/RimWorld by Ludeon Studios/Config",
-            f"{expanduser('~')}/.steam/debian-installation/steamapps/workshop/content/294100",
-        ]
+
```

Comment:
This is fine by me. I approve of these changes for now, however eventually we are going to build something that will properly dynamically detect the paths from a base Steam prefix, etc. Nice work and thank you for taking the initiative!